code/modules/food_and_drinks/food/customizables.dm

```diff
@@ -294,7 +294,7 @@
 	icon_state = "bowl"
 	reagent_flags = OPENCONTAINER
 	materials = list(MAT_GLASS = 500)
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_SMALL
```

Comment:
A better long term solution would be adding a "size whitelist" variable to the storage component for typecaches of items that can be inserted regardless of size and then whitelist bowls and food reagent containers for the tray.